tests/processes/LogPrinterThreadTest.py

```diff
@@ -2,14 +2,10 @@
 import unittest
 
 from coala_utils.ContextManagers import retrieve_stdout
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.processes.LogPrinterThread import LogPrinterThread
 
 
-class TestPrinter(LogPrinter):
-
-    def __init__(self):
-        LogPrinter.__init__(self, self)
+class TestPrinter():
```

Comment:
You can remove the log_printer kwarg from the LogPrinterThread class, which is only used in execute_section, which in turn is only used in run_coala. The run_coala function is never called with a log_printer kwarg -> use logging module in LogPrinterThread class